packages/packages/api/src/medical/client/metriport.ts

```diff
@@ -328,27 +328,20 @@ export class MetriportMedicalApi {
     return documentQuerySchema.parse(resp.data).queryStatus;
   }
 
-  // TODO #435 review the return type of this function
   /**
-   * Returns document references for the given patient across HIEs.
-   * Usually called after a successful call to `getDocuments`, which provides
-   * a Document location.
+   * Returns presigned url for document in s3
    *
-   * @param patientId Patient ID for which to retrieve document metadata.
-   * @param facilityId The facility providing the NPI to support this operation.
-   * @param location The location of the document.
-   * @return The document's contents (bytes).
+   * @param fileName The file name of the document in s3.
+   * @return presigned url
    */
   //eslint-disable-next-line @typescript-eslint/no-explicit-any
-  async getDocument(patientId: string, facilityId: string, location: string): Promise<any> {
+  async getDocument(fileName: string): Promise<{ url: string }> {
     const resp = await this.api.get(`${DOCUMENT_URL}/download`, {
```

Comment:
perhaps this can be `getDocumentUrl` & `/downloadUrl` now?

also, the docs will need updating when we agree on an approach

Reply:
This makes sense to me